search-commons/src/main/java/no/unit/nva/search2/ticket/TicketParameter.java

```diff
@@ -95,7 +95,7 @@ public enum TicketParameter implements ParameterKey {
     PUBLICATION_OWNER_NOT(FUZZY_KEYWORD, NOT_ONE_ITEM, PUBLICATION_OWNER_KEYWORD),
     PUBLICATION_STATUS(KEYWORD, ONE_OR_MORE_ITEM, PUBLICATION_STATUS_KEYWORD),
     PUBLICATION_STATUS_NOT(KEYWORD, NOT_ONE_ITEM, PUBLICATION_STATUS_KEYWORD),
-    PUBLICATION_TITLE(FUZZY_TEXT, ALL_ITEMS, PUBLICATION_MAIN_TITLE_KEYWORD, PHI),
+    PUBLICATION_TITLE(TEXT, ALL_ITEMS, PUBLICATION_MAIN_TITLE_KEYWORD, PHI),
```

Comment:
Are you sure this PR is only for documentation?

Reply:
FUZZY_TEXT is the same as TEXT, i.e. it is not in use anymore

Reply:
I removed it, so that I don't have to document it.